devenv/docker/blocks/tempo/docker-compose.yaml

```diff
@@ -64,13 +55,16 @@ services:
       JAEGER_SAMPLER_PARAM: 1
     labels:
       namespace: tns
-    logging: *default-logging
+    logging: 
+      driver: loki
+      options:
+        loki-url: 'http://localhost:3100/api/prom/push'
+        labels: namespace
 
   tempo:
-    image: grafana/tempo:main-9a8474f
+    image: grafana/tempo:main-dcf8a2a
```

Comment:
Is there a reason for pinning this instead of using some version tag or latest?

Reply:
We don't want to always be using the latest at the time this docker-compose is run, as Tempo updates the image  which may break the devenv script when new settings are added/updated/removed. Safer to stick to a version which we know works with the settings (tempo.yaml etc) we have.